include/dav1d/picture.rs

```diff
@@ -102,44 +109,184 @@ pub struct Dav1dPicture {
     pub allocator_data: Option<NonNull<c_void>>,
 }
 
-type AlignedPixelChunk = Align64<[u8; RAV1D_PICTURE_ALIGNMENT]>;
+#[derive(Clone, FromZeroes, FromBytes, AsBytes)]
+#[repr(C, align(64))]
+pub struct AlignedPixelChunk([u8; RAV1D_PICTURE_ALIGNMENT]);
+
 const _: () = assert!(mem::align_of::<AlignedPixelChunk>() == RAV1D_PICTURE_ALIGNMENT);
 const _: () = assert!(mem::size_of::<AlignedPixelChunk>() == RAV1D_PICTURE_ALIGNMENT);
 
-pub struct Rav1dPictureDataComponent {
+pub struct Rav1dPictureDataComponentInner {
+    /// A ptr to the start of pixels, even if [`Self::stride`] is negative.
```

Comment:
It's a little bit complex, since the buffer can have padding, and can be shared/split among multiple `Rav1dPictureDataComponentInner`s.  It is to the start of pixels that we are allowed to access.

Reply:
I tried to explain this better in [`7cfc0c9..9bb9b2f`](https://github.com/memorysafety/rav1d/compare/7cfc0c95170e5fc01d265da9bb3e522f6f6fc382..9bb9b2fa7147263ae12d7dcbbf2a24cf7aba4c04).  Is that better?  Not really sure how to explain it well.